src/zenml/container_registries/__init__.py

```diff
@@ -39,9 +39,6 @@
 from zenml.container_registries.gcp_container_registry import (
     GCPContainerRegistry,
 )
-from zenml.container_registries.github_container_registry import (
-    GitHubContainerRegistry,
```

Comment:
Is this removed by accident here?

Reply:
I moved the github container registry into the integration (as the integration initially didn't have any requirements and was installed by default). Now I need one pip requirement for the secret encryption, do you think I should move the container registry back into default ZenML?